releases/1.8/beta/bundle.yaml

```diff
@@ -0,0 +1,253 @@
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+    options:
+      kind: ingress
+  istio-pilot:
+    charm: istio-pilot
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+    options:
+      default-gateway: kubeflow-gateway
+  jupyter-controller:
+    charm: jupyter-controller
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: notebook-operators
+  jupyter-ui:
+    charm: jupyter-ui
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: notebook-operators
+  katib-controller:
+    charm: katib-controller
+    channel: latest/beta
+    scale: 1
+    _github_repo_name: katib-operators
+  katib-db:
+    charm: mysql-k8s
+    channel: 8.0/stable
+    scale: 1
+    trust: true
+    constraints: mem=2G
+  katib-db-manager:
+    charm: katib-db-manager
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: katib-operators
+  katib-ui:
+    charm: katib-ui
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: katib-operators
+  kfp-api:
+    charm: kfp-api
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-db:
+    charm: mysql-k8s
+    channel: 8.0/stable
+    scale: 1
+    trust: true
+    constraints: mem=2G
+  kfp-persistence:
+    charm: kfp-persistence
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-profile-controller:
+    charm: kfp-profile-controller
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-schedwf:
+    charm: kfp-schedwf
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-ui:
+    charm: kfp-ui
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-viewer:
+    charm: kfp-viewer
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-viz:
+    charm: kfp-viz
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  knative-eventing:
+    charm: knative-eventing
+    channel: latest/beta
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-eventing
+    _github_repo_name: knative-operators
+  knative-operator:
+    charm: knative-operator
+    channel: latest/beta
+    scale: 1
+    trust: true
+    _github_repo_name: knative-operators
+  knative-serving:
+    charm: knative-serving
+    channel: latest/beta
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-serving
+      istio.gateway.namespace: kubeflow
+      istio.gateway.name: kubeflow-gateway
+    _github_repo_name: knative-operators
+  kserve-controller:
+    charm: kserve-controller
+    channel: latest/beta
+    scale: 1
+    trust: true
+    options:
+      deployment-mode: serverless
```

Comment:
kserve-controller deploys with serverless mode by default, no need to specify here.

Reply:
good catch, removing it. Note that we have this in latest/beta and latest/edge too.